File: Samples/Scripts/Spark/Python/GettingStarted/TextSearch.py
from pyspark import SparkContext

# Read the input file 
text_file = sc.textFile("/Data/NASA_Access_Log")

# Filters the input lines that contains the Internet Address "lya.colorado.edu"
errors = text_file.filter(lambda line: "lya.colorado.edu" in line)

# Saves the filtered lines in text file
errors.saveAsTextFile("/Data/Output/TextSearchPython")

File: Samples/Scripts/Spark/Scala/GettingStarted/TextSearch.scala
//Textsearchapplication: Filters the lines contains "lya.colorado.edu" from input 'NASA_Access_Log'
val textFile = sc.textFile("/Data/NASA_Access_Log")
    
val errors = textFile.filter(line => line.contains("lya.colorado.edu"))

errors.collect().foreach(println)